Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -52,7 +52,7 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 * `--cephfs-metadata-pool-name`: (optional) Provides the name of the cephfs metadata pool
 * `--cephfs-filesystem-name`: (optional) The name of the filesystem, used for creating CephFS StorageClass
 * `--cephfs-data-pool-name`: (optional) Provides the name of the CephFS data pool, used for creating CephFS StorageClass
-* `--rados-namespace`: (optional) Divides a pool into separate logical namespaces, used for creating RBD PVC in a RadosNamespaces
+* `--rados-namespace-name`: (optional) Divides a pool into separate logical namespaces, used for creating RBD PVC in a RadosNamespaces
```

Comment:
Why is this parameter renamed? It seems better without the `-name` suffix.

Reply:
@Madhu-1 any thoughts, can you confirm should we revert it??

Reply:
I was trying to keep all the flags similar like pool-name filesystem-name metdata-pool-name etc, no objections to revert it back as well

Reply:
I thought we ate adding the support for radosnamespace in this PR, if we already have support for it with existing flag we should not change anything in flag as it breaks the backwards compatibility